repair_purchase_return/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+The purpose of this module is to modify the behaviour of the repair standard module. Achieving more flexibility in a repair process.
+
+The first change that it applies is the modification of the creation of stock moves in the repair flow:
+
+* Confirm Repair: creates draft stock moves on confirmation of the repair order.
+* Start Repair: confirms the stock moves.
+* End Repair: completes the stock moves and ends the repair order.
+
+Another feature which this module includes is that it allows to add components during all the different stages of the repair process, which adjusts better to the needs of some businesses.
```

Comment:
This Description, as well as the Usage, don't match with the module's functionality (they are from repair_stock_move module). Could you please update them?

Reply:
Updated